### DISCUSSION TITLE
.NET bindings ?

### DISCUSSION THREAD

**Discussion by goswinr** [elalish/manifold#551] [Q&A]
Thank you for creating this library!
I would like to use it from F#. Do you know if there are any .NET bindings available? 
My C++ knowledge is poor. I understand I would have to create a managed CLI/C++ wrapper DLL with the static lib files compiled on Windows.

**Comment by geoffder** [comment]
We provide [C bindings](https://github.com/elalish/manifold/tree/master/bindings/c) that are tailored for use via FFI from GC languages that you might find useful.

**Comment by goswinr** [comment]
Ah! Thanks ! How can I compile these bindings to a DLL on Windows so that can call them via [ P/Invoke ](https://learn.microsoft.com/en-us/dotnet/standard/native-interop/pinvoke) ?

**Reply by geoffder** [reply]
I'm afraid I can't help you with anything windows, or specific to dotnet. I'm sure there are plenty of resources out there on using C bindings though.

**Reply by pca006132** [reply]
See https://github.com/elalish/manifold/blob/master/.github/workflows/manifold.yml#L151. You also need to add `-DMANIFOLD_CBIND=ON` to the cmake command.
